jax/_src/scipy/signal.py

```diff
@@ -637,3 +637,33 @@ def istft(Zxx: Array, fs: ArrayLike = 1.0, window: str = 'hann',
 
   time = jnp.arange(x.shape[0], dtype=np.finfo(x.dtype).dtype) / fs
   return time, x
+
+@_wraps(osp_signal.hilbert)
+def hilbert(x: Array, N: int = None, axis: int = -1):
```

Comment:
`N` needs to be `Optional[int]`

Reply:
I like `Optional[int]` too, lets just go with that

Reply:
`Optional[int]` is more idiomatic